spec/jobs/moab_replication_audit_job_spec.rb

```diff
@@ -45,12 +45,11 @@
 
     it 'calls PartReplicationAuditJob once per related endpoint' do
       new_target_endpoint = create(:zip_endpoint)
-      new_nontarget_endpoint = create(:zip_endpoint,
-                                      preservation_policies: [create(:preservation_policy, preservation_policy_name: 'giant_datasets_policy')])
+      new_nontarget_endpoint = create(:zip_endpoint)
```

Comment:
i think we want to tweak this test just a bit more.  since there's no longer a notion of druid-specific target endpoints, i'd rename `new_target_endpoint` and `new_nontarget_endpoint` to something like `new_target_endpoint_1` and `new_target_endpoint_2`, keep your loop change on line 52, and get rid of what's now line 55 (since i think `PartReplicationAuditJob.perform_later` does get called on that, but the expectation in the loop on line 53 covers it's one invocation?).

Reply:
Done.